charts/metallb/Chart.yaml

```diff
@@ -19,9 +19,11 @@ type: application
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
+# NOTE: this value is updated by the metallb release process
```

Comment:
Thanks, @gclawes .  This is a nit for the future, but maybe we should change "0.9.6" to something else, like "0.0.0" ... I just thought of this, but now this version in main is behind what's in the v0.10 branch, since that automation just updates the release branch.

Reply:
Yeah, that makes sense.